File: Models/PromptItem.vb
Namespace Models

    ''' <summary>
    ''' Lưu thông tin một prompt với ID cố định
    ''' </summary>
    Public Class PromptItem
        Public Property Id As Integer
        Public Property Name As String
        Public Property Content As String

        Public Sub New()
        End Sub

        Public Sub New(id As Integer, name As String, content As String)
            Me.Id = id
            Me.Name = name
            Me.Content = content
        End Sub

        ''' <summary>
        ''' Trả về tên hiển thị dạng "STT. Tên"
        ''' </summary>
        Public ReadOnly Property DisplayName As String
            Get
                Return String.Format("{0}. {1}", Me.Id, Me.Name)
            End Get
        End Property
    End Class

End Namespace

File: Models/PromptItem.cs
namespace Subtitle_draft_GMTPC.Models
{
    /// <summary>
    /// Lưu thông tin một prompt với ID cố định
    /// </summary>
    public class PromptItem
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Content { get; set; }

        public PromptItem()
        {
        }

        public PromptItem(int id, string name, string content)
        {
            Id = id;
            Name = name;
            Content = content;
        }

        /// <summary>
        /// Trả về tên hiển thị dạng "STT. Tên"
        /// </summary>
        public string DisplayName => $"{Id}. {Name}";
    }
}
